### PR TITLE
javascript I

### PR DESCRIPTION
**Pull request by claridiva2000**

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -92,17 +92,32 @@ console.log(carModels);
...
+  }
+}
+console.log(carYears);
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars =[];
-console.log(); 
+for(let i=0; i<inventory.length; i++) {
+  if(inventory[i].car_year <= 2000){
```
Comment (JoeBugajski) [inline assignments/arrays.js:106]: Very close! But if we only want cars _older than_ 2000, we'd just use this:

```suggestion
  if(inventory[i].car_year < 2000){
```

**Review by JoeBugajski — APPROVED** [review]
Very close! Nice work.

**Comment by JoeBugajski** [comment]
Are you remembering to branch _before_ you make any commits? I only ask because your commits for objects.js are not on this pull request!

**Comment by claridiva2000** [comment]
I fixed the branch issue last night.  Probably after you reviewed it.
Should be under jasmine_logan.

**Pull request merged**